src/ads/ima/manager.js

```diff
@@ -244,36 +250,36 @@ Scoped.define("module:Ads.IMA.AdsManager", [
 
             __events: function() {
                 return [
-                    google.ima.AdErrorEvent.Type.AD_ERROR,
-                    google.ima.AdEvent.Type.AD_CAN_PLAY,
-                    google.ima.AdEvent.Type.IMPRESSION,
+                    google.ima.AdErrorEvent.Type.AD_ERROR, // adError
```

Comment:
added these so we can global string search event names and find that they came from here!